### ISSUE TITLE
Staging Review finding: Appointment confirmation not announced for screen readers

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#111491]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Unified Appointment Experience
**Product:** Appointments
**Feature:** Community Care Direct Scheduling

## Findings details
**VA.gov Experience Standard - issue:** [User receives appropriate notification from the VA or VA.gov about the status of their request.](https://design.va.gov/about/experience-standards/#user-receives-appropriate-notification-from-the-va-or-vagov-about-the-status-of-their-request)
**VA.gov Experience Standard - category:** [Credibility](https://design.va.gov/about/experience-standards/#category-credibility)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

When an appointment is confirmed, the user is taken to the appointment confirmation page and focus is placed on the `h1`. However, the text that says the appointment has been scheduled is below that.

## Recommended action

Update the `h1` to contain the text that the appointment has been scheduled.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG21/#status-messages)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
